src/sources/docker_logs/mod.rs

```diff
@@ -1166,7 +1166,7 @@ impl ContainerLogInfo {
                         LogNamespace::Legacy => {
                             partial_event_merge_state.merge_in_next_event(
                                 log,
-                                &[log_schema().message_key().to_string()],
+                                &[log_schema().message_key().unwrap().to_string()],
```

Comment:
Am I right to presume this `unwrap` is safe because of the legacy namespace? It might be helpful to document this with a comment or by using `expect` instead (here and below).

Reply:
Replaced with `expect(..)`. Yes, message key should not be `None` at this point and (same goes for other instances in this file).